.travis.yml

```
@@ -12,6 +12,11 @@ rvm:
   - '2.5'
   - '2.6'
   - '2.7'
+  - 'truffleruby-head'
+matrix:
```

`matrix` is an old name for `jobs`. (There's a validation message: 
    <code>root</code>: key <code>matrix</code> is an alias for <code>jobs</code>, using <code>jobs</code> in the build validation config.)
  

```suggestion

jobs:
```